src/app/helpers/googleService.js

```diff
@@ -0,0 +1,67 @@
+const envHelper = require('./environmentVariablesHelper.js');
+const CONSTANTS = require('./constants');
+const _ = require('lodash');
+const httpSatusCode = require('http-status-codes');
+const logger = require('sb_logger_util_v2');
+const {sendRequest} = require('./httpRequestHandler');
+const uuidv1 = require('uuid/v1');
+const {parseJson} = require('./utilityService');
+
+
+const validateRecaptcha = async (req, res) => {
+  let errType;
+  try {
+    if (!req.query.captchaResponse) {
+      errType = 'MISSING_QUERY_PARAMS';
+      throw new Error('MISSING_CAPTCHA_RESPONSE');
+    }
+    errType = 'ERROR_VALIDATING_CAPTCHA';
+    const queryParams = '?secret=' + envHelper.google_captcha_private_key + '&response=' + req.query.captchaResponse;
+    const options = {
+      method: CONSTANTS.HTTP.METHOD.POST,
+      url: CONSTANTS.GOOGLE_VERIFICATION_URL + queryParams
+    };
+    let responseData = await sendRequest(options);
+    errType = 'ERRORED_PARSING_JSON';
+    responseData = parseJson(responseData);
+    errType = '';
+    if (responseData && responseData.success) {
+      logger.info({msg: 'googleService:validateRecaptcha success', data: responseData});
+      res.status(httpSatusCode.OK).send(responseData)
+    } else {
+      logger.info({msg: 'googleService:validateRecaptcha errored', data: responseData, errType: errType});
+      throw new Error('CAPTCHA_VALIDATING_FAILED');
+    }
+  } catch (error) {
+    logger.error({
```

Comment:
Log userID, dId getting in the request headers. This is the key information to identify who are these users.